<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrequencyDomain</class>
 <widget class="QWidget" name="FrequencyDomain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1001</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>频域波形展示</string>
   </property>
   <widget class="QCustomPlot" name="MyCustomPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>981</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>871</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>材料寿命预估</string>
   </property>
   <widget class="QCustomPlot" name="MyCustomPlotNext" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>851</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="tbnDatabase">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>680</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>数据库</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbnTimeDomain">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>320</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>时域</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbnChangeWaveStyle">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>220</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>切换</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFundamentalWave">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>390</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>2M基波A1=</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>4M二次谐波A2=</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSecondWave">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>390</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>390</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>非线性参量β=</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNolinearPara">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>390</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="tbnReadAndCalculate">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>170</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leMaterialNumber">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>460</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>460</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>材料编号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leMaterialType">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>500</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>500</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>材料类型：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leHarmType">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>540</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="le_2">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>540</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>损伤类型：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTestCondition">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>580</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="le">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>580</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>试验条件：</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbnInquiry">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>620</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbnConfirmHarm">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>620</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>定标</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbnSaveData">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>680</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbnStartDraw">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbnClearWaves">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>270</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbnLoadTimeData">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbnSaveFreData">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>120</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
